paper.md

```diff
@@ -70,7 +70,7 @@ to conduct post-disaster assessment using large-scale mobility data. The toolkit
 allows the user to conduct pre-processing of data, validation of the data 
 representativeness, home and office location estimation, post-disaster displacement analysis, 
 and point-of-interest visit analysis. The purpose of `Mobilkit` is to provide urban planners, 
-disaster policy makers and researchers an easy-to-use and practical toolkit to visualize, 
+disaster policy makers, and researchers an easy-to-use and practical toolkit to visualize, 
 analyze, and monitor post-disaster disruption and recovery. The software is freely-available 
 on GitHub along with online documentation and Jupyter Notebooks that provides step-by-step tutorials.

```

Comment:
Add a link to the repository here.

Reply:
Will add it in the merged version and mention the issue.